app/assets/javascripts/options.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
The coffee files are auto-generated, but we need to replace them later if they add JS. Can you delete them?